cincinnati/src/plugins/mod.rs

```diff
@@ -425,8 +432,12 @@ where
         let tx = tx.clone();
 
         std::thread::spawn(move || {
-            let io_future =
-                async { tokio::time::timeout(timeout, process(plugins, initial_io)).await };
+            let span = get_tracer().start("plugins", None);
```

Comment:
Could we move this to `fn process` further up at https://github.com/openshift/cincinnati/pull/266/files#diff-2c51cb19b5fdc6531cb418780c1691c9R386?